src/main/java/com/everis/entity/Car.java

```diff
@@ -56,6 +56,15 @@
 	public Car() {
 		super();
 	}   
+	
+	public Car(@NotNull(message = "Brand can't be null") String brand, Date registration,
```

Comment:
Why you've added the bean validations annotations here ?

Reply:
i just needed i constructor with parameters so i generated by eclipse shortcuts